rpc/connection.go

```diff
@@ -286,8 +286,7 @@ func copyTLSConfig(c *tls.Config) *tls.Config {
 	if c == nil {
 		return nil
 	}
-	tmp := *c
-	return &tmp
+	return c.Clone()
```

Comment:
`tls.Config` has a `sync.Mutex` field.  I think `Clone()` was added in a recent release.